setup.py

```diff
@@ -1,4 +1,5 @@
 import setuptools
+from mabwiser._version import __author__, __version__
```

Comment:
IIRC setting a version like this isn't encouraged. As you mentioned before, depending on how your PYTHONPATH is configured this might import the already installed mabwiser, which will have an old version.

I have a [workaround](https://github.com/fidelity/textwiser/blob/master/setup.py#L10-L11) in TextWiser, but you might be able to find something cleaner than that.

Reply:
Thanks for the heads up! It worked correctly when I tested it, but I'll try the workaround you have

Reply:
(If it turns out too complicated/involved, might wanna consider fixing version update a separate PR than LSH) 

Could you also comment once this version changes are in place in some way, like this or Doruk's workaround, what's the suggested procedure for the next person bumping version? Business, as usual, is to browse a few files and update the hardcoded values manually (and forget some and to pray the reviewers catch it). What's the process now?

Reply:
We now only need to increment the version number in _version.py